### PR TITLE
Testing - manifestv2

### PR DESCRIPTION
**Pull request by Salamandar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Salamandar** [comment]
!testme

**Comment by Salamandar** [comment]
!testme

File: conf/nginx.conf [hunk 1]
```diff
@@ -2,22 +2,22 @@
...
 }
 
-location = __PATH__/ {
-  default_type text/plain;
-  return 200 "This is where Standard Notes - Syncing Server is installed.";
-}
+# location = __PATH__/ {
+#   default_type text/plain;
+#   return 200 "This is where Standard Notes - Syncing Server is installed.";
+# }
```
Comment (alexAubin) [inline conf/nginx.conf:15]: Well uh this seemed important to have the test pass ?

**Comment by ericgaspar** [comment]
!testme

File: conf/nginx.conf (hunk 1 above)
Comment (Salamandar) [reply conf/nginx.conf]: Yes, uh, can’t remember why i commented this out.

**Pull request merged**